src/runtime/config/configuration-clh.toml.in

```diff
@@ -238,6 +231,11 @@ disable_guest_seccomp=@DEFDISABLEGUESTSECCOMP@
 # See: https://pkg.go.dev/github.com/kata-containers/kata-containers/src/runtime/virtcontainers#ContainerType
 sandbox_cgroup_only=@DEFSANDBOXCGROUPONLY@
 
+# If enabled, the runtime will attempt to determine appropriate sandbox size (memory, CPU) before booting the virtual machine. In
+# this case, the runtime will not dynamically update the amount of memory and CPU in the virtual machine. This is generally helpful
+# when a hardware architecture or hypervisor solutions is utilized which does not support CPU and/or memory hotplug.
```

Comment:
I can add that for the kubernetes case, but this will also be relevant in the non-kubernetes scenario (when launching with a tool _like_ ctr)

Reply:
Yep, add a note about `ctr` as usual, if possible.

What I'm trying to avoid here is a bunch of issues opened because someone tries to use something and has absolutely no information on whether that should work or not with the setup they have (eg: k8s, containerd, cri-o, ctr, ...).   As the matrix of elements is not **that** huge (only k8s, containerd, cri-o, and ctr) I think it's worth mentioning.

Sounds reasonable?

Reply:
updated, PTAL.